pkg/sharedconf/sharedconf.go

```diff
@@ -60,6 +62,14 @@ func Decode(v interface{}) error {
 		sharedConf.JWTSecret = "changemeplease"
 	}
 
+	if sharedConf.MetricsDataDriverType == "" {
```

Comment:
Also, remove setting the defaults here. If it's not set, then don't serve the metrics. This provides a way to disable these.